endpoints-support.md

```diff
@@ -17,268 +17,268 @@
-|<sub>/rest/logical-interconnects/{id}/telemetry-configurations/{tcId}</sub>              | PUT       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
-|<sub>/rest/logical-interconnects/compliance</sub>                                        | PUT      | :heavy_multiplication_x:       | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: |:heavy_multiplication_x: |:heavy_multiplication_x:   | :heavy_multiplication_x: |:heavy_multiplication_x: |:heavy_multiplication_x: |:heavy_multiplication_x: |:heavy_multiplication_x: |
-|<sub>/rest/logical-interconnects/{id}</sub>                                              | PATCH     | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:  | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |
-|<sub>/rest/logical-interconnects/bulk-inconsistency-validation</sub>                     | POST      | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:  | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/logical-interconnects</sub>                                                   | GET       | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |:white_check_mark:  |:white_check_mark:   |
+|<sub>/rest/logical-interconnects/locations/interconnects</sub>                           | POST      | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:white_check_mark:  |:white_check_mark:   |
+|<sub>/rest/logical-interconnects/locations/interconnects</sub>                           | DELETE    | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:white_check_mark:  |:white_check_mark:   |
```

Comment:
change to heavy_minus for api2400 and 2600

Reply:
Done